source/uwp/Renderer/lib/XamlBuilder.cpp

```diff
@@ -616,10 +622,272 @@ namespace AdaptiveCards { namespace XamlCardRenderer
         *uiShowCard = localUiShowCard.Detach();
     }
 
+    void SerializeTextInput(
+        Json::Value& jsonValue,
+        const char * idString,
+        InputItem* inputItem)
+    {
+        ComPtr<ITextBox> textBox;
+        THROW_IF_FAILED(inputItem->uiInputElement.As(&textBox));
+
+        HString text;
+        THROW_IF_FAILED(textBox->get_Text(text.GetAddressOf()));
+
+        std::string textString;
+        if (text.IsValid())
+        {
+            THROW_IF_FAILED(HStringToUTF8(text.Get(), textString));
+        }
+
+        jsonValue[idString] = textString.c_str();
+    }
```

Comment:
All the serializeXInput methods feel like they should be part of the input classes. What do you think?